.gitignore

```diff
@@ -3,3 +3,6 @@ node_modules
 cert
 database
 logs
+
+# JetBrains IDEs
```

Comment:
Also, what is this addition in `.gitignore` used for? Can this be kept on your individual fork?

Reply:
Anybody contributing to this project with a JetBrains IDE (e.g. IntelliJ IDEA Ultimate, WebStorm) will benefit from having this line here. If this line isn't here, they all have to add it to `.git/info/exclude` manually.